src/main/java/com/courtcanva/ccfranchise/controllers/StaffController.java

```diff
@@ -0,0 +1,37 @@
+package com.courtcanva.ccfranchise.controllers;
+
+import com.courtcanva.ccfranchise.dtos.StaffVerifyEmailPostDto;
+import com.courtcanva.ccfranchise.exceptions.ExpiredVerificationTokenException;
+import com.courtcanva.ccfranchise.services.StaffService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@Slf4j
+@RestController
+@RequestMapping("/staff")
+@RequiredArgsConstructor
+public class StaffController {
+
+    private final StaffService staffService;
+
+    @PostMapping("/verification")
+    @ResponseStatus(HttpStatus.CREATED)
+    public void sendVerificationEmail(@RequestParam Long id) {
```

Comment:
since you use id from url instead of body, I think no need to use post method, better use get method.
And also, post is for creating object, there is no creation in verification stage.

Reply:
There is actually, verification token was created in this process.